src/chrome/content/rules/Dell.xml

```diff
@@ -87,7 +68,6 @@
 	<target host="dell.com" />
 	<target host="www.dell.com" />
 	<target host="accessories.dell.com" />
-	<target host="afcs.dell.com" />
```

Comment:
WFM

Reply:
A bunch of Google-indexed pages redirect to HTTP, e.g. http://afcs.dell.com/support/home/.